tools/api-markdown-documenter/src/api-item-transforms/helpers/Helpers.ts

```diff
@@ -533,6 +533,7 @@ export function createDeprecationNoticeSection(
 export function createExamplesSection(
 	apiItem: ApiItem,
 	config: Required<ApiItemTransformationConfiguration>,
+	title: string = "Examples",
```

Comment:
Nit: Can we call this `headingText` or something like that? That's more in line with the terminology used elsewhere in this helper suite. `wrapInSection` uses "title", but it has more particular semantics.